menu.rb

```diff
@@ -0,0 +1,14 @@
+def display_menu
+  puts "\nWelcome to School Library App!"
+  puts "\nChoose an option by entering an alphabet:"
```

Comment:
Great job My boss! Bravo.
On line 2, please consider changing the texts from (Welcome to School Library App!") to (Welcome to Catalog Of My Things App!") so that it would align with the name of the App.